### PR TITLE
Add jquery-advanced-news-ticker@1.0.1 w/ git auto-update

### LINKED ISSUE #10020: [Request] Add newsTicker
**Library name:** JQuery Advanced News Ticker 
**Git repository url:**https://github.com/risq/jquery-advanced-news-ticker
**npm package url(optional):** 
**License(s):**GNU GENERAL PUBLIC LICENSE

**Official homepage:**http://risq.github.io/jquery-advanced-news-ticker/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sajochiu**
w/ git auto-update, close #10020, cc @risq

@x09326 Please help me review this.

Pull request for issue: #10020
Related issue(s): #10020 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/risq/jquery-advanced-news-ticker
 * Official website (optional, not the repository):https://risq.github.io/jquery-advanced-news-ticker/
 * NPM package url (optional):
 * License and its reference:GPL-2.0 https://github.com/risq/jquery-advanced-news-ticker/blob/master/LICENSE.txt
 * GitHub / Bitbucket popularity (required):
   - Count of stars:263
   - Count of watchers:20
   - Count of forks:106
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by x09326** [comment]
@cdnjs/intern2 please help me review this pull requset, thanks.

File: ajax/libs/jquery-advanced-news-ticker/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/risq/jquery-advanced-news-ticker.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/risq/jquery-advanced-news-ticker.git",
+    "fileMap": [
+      {
+        "basePath": "js",
```
Comment (pvnr0082t) [inline ajax/libs/jquery-advanced-news-ticker/package.json]: The latest master branch of [jquery-advanced-news-ticker](https://github.com/risq/jquery-advanced-news-ticker.git) put the `newsTicker.js` at the root of repository. Let's use `""` as basePath.

File: ajax/libs/jquery-advanced-news-ticker/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        "files": [
+          "newsTicker*"
```
Comment (pvnr0082t) [inline ajax/libs/jquery-advanced-news-ticker/package.json]: The latest master branch of [jquery-advanced-news-ticker](https://github.com/risq/jquery-advanced-news-ticker.git) seems rename the file to `jquery.newsTicker.js` and `jquery.newsTicker.min.js`. Let's use `"*newsTicker*"` here.

**Comment by pvnr0082t** [comment]
@x09326 Just notice that the file structure and naming scheme of released versions are different from the latest master branch.
cc @PeterDaveHello @sajochiu

**Comment by x09326** [comment]
@pvnr0082t 
I know the file structure and naming scheme of released versions are different from the latest master branch.
But the auto-update only download the version branch right?
cc @PeterDaveHello

**Comment by PeterDaveHello** [comment]
@x09326 Version is from tag, tagged commit, not related with any of branch things.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by extend1994** [comment]
I noticed a keyword `(deprecated)` showed in its repo. I think the possibility of that the author release a new version is low and the revised auto-update config can get the files in current master branch. So I update a few parts in this PR to help this PR being merged.
1. Rebase the branch
2. Update `filename` from `newsTicker.min.js` to `js/newsTicker.min.js`
3. Update the auto-update from
```javascript
      {
        "basePath": "js",
        "files": [
          "newsTicker*"
        ]
      }
```

to
```javascript
      {
        "basePath": "",
        "files": [
          "*newsTicker*"
        ]
      }
```

@cdnjs/intern3 @cdnjs/library-reviewer @cdnjs/active-contributor Please help review this, thank you all.

**Comment by sufuf3** [comment]
@cdnjs/active-contributor @hare1039 @qwe19272375 Please help review this, @extend1994 helped modify this PR.
Thanks.

**Comment by PeterDaveHello** [comment]
@extend1994 I believe the complexity here worth you to send a new PR to close this one to keep your credit next time!

**Pull request merged**